autorest/code_generator.py

```diff
@@ -94,7 +94,7 @@ def _create_code_model(self, yaml_code_model):
         namespace = self._autorestapi.get_value("namespace")
         _LOGGER.debug("Namespace parameter was %s", namespace)
         if not namespace:
-            namespace = get_namespace_name(yaml_code_model["info"]["title"])
+            namespace = yaml_code_model["info"]["title"]
```

Comment:
why don't we need the "get_namespace_name" anymore?

Reply:
thanks, should call from ['info']['python_title']